modules/ROOT/pages/dataweave-flow-control-precedence.adoc

```diff
@@ -1,55 +1,108 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 
-DataWeave expressions are compiled in a specific order. The result of compiling something at one level may be used as an input for expressions in higher levels, but not vice-versa. Expressions are ordered in the table from first compiled to last.
+DataWeave expressions are compiled in a specific order. The result of
+a compilation of something at one level can serve as input for expressions
+in higher levels, but not at lower levels.
 //LINK TO DW 1.0 LANDING PAGE IN MULE 3.9 DOCS:
 include::partial$dataweave1-links.adoc[tag=dataweave1LandingPage]
 
-[%header,cols="30a,60a,10a"]
+The following table orders operators and functions from first compiled (1) to
+last compiled (10).
+
+[%header,cols="1,8"]
 |===
-| Operator
-| Description
 | Level
+| Operator or Function
 
-| `using`, all unary operators
-|
 | 1
+a|
+* `using`: For initializing local variables in a DataWeave script.
+* Unary DataWeave operators at this level:
+** `.^`: Schema selector.
+** `.#`: Namespace selector.
+** `..`: Descendants selector.
+** `not`: Logical operator for negation.
+** `.@`: All attribute selector, for example, in a case that uses the
+expression `payload.root.a.@` to return the attributes and values of the
+input payload `<root> <a attr1="foo" attr2="bar" /></root>` within the
+array of objects `{ "attr1": "foo", "attr2": "bar" }`.
 
-| `as`
-| Type coercion expression
 | 2
+a|
+* `as`: Used for type coercion.
 
-| `&#42;` `/`
-| Multiplication and division
 | 3
+a|
+* `&#42;`: Multiply.
+* `/`: Divide.
 
-| `+ - >> <<`
-| Addition, subtraction, and binary right and left shift operators
 | 4
+a|
+* `+`: Add.
+* `-`: Subtract.
+* `>>`: Prepend data.
+* `<<`: Append data.
 
-| `&gt;= &lt;= < > is`
-| Relational / Type Comparison
 | 5
+a|
+* `&gt;`: Greater than or equal to.
+* `=`: Equal to.
+* `&lt;`: Less than or equal to.
+* `<`: Less than.
+* `>`: Greater than.
+* `is`: Compare type, for example, used in
+xref:dataweave-cookbook-remove-certain-xml-attributes.adoc[Remove Specified XML Attributes].
 
-| `!= ~= ==`
-| Equality evaluators
 | 6
+a|
+* `!=`: Not equal to.
+* `~=`: Contains.
+* `==`: Equality operator that tries to coerce one value to the type of the
+other when the types are different.
 
-| AND
-| Conditional And
 | 7
+a|
+* `and`: Logical `and` operator.
 
-| OR
-| Conditional OR
 | 8
+a|
+* `or`: Logical `or` operator.
 
-| default, <<Pattern Matching>>, matches, map, mapObject, groupBy, filter            | Default Value / Pattern Matching / Binary Operators
 | 9
+a|
+* `default`: For setting a default value.
+* `case` and `else`: For pattern matching.
+* DataWeave functions at this level:
+** `matches`
+** `match`
+** `map`
+** `mapObject`
+** `groupBy`
+** `filter`
+* Binary DataWeave operators at this level:
+** `.@keyName` and `.*@`: Binary DataWeave attribute selectors.
+** `.*`: Multi-value selector.
+** `.^`: Metadata selector.
+** `.&`: Object key filter.
```

Comment:
This is a Key Value Pair selector

Reply:
changed "Object key filter" to "Key-value pair selector"